yt_dlp/extractor/pandavideo.py

```diff
@@ -0,0 +1,76 @@
+from .common import InfoExtractor
+
+
+class PandaVideoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?P<server>[\w-]+)\.tv\.pandavideo\.com\.br/embed/?\?(?:[^#]*&)?v=(?P<id>[\da-f-]+)'
+    _EMBED_REGEX = [rf'<iframe[^>]+\bsrc=[\'"](?P<url>{_VALID_URL})']
```

Comment:
just realized we need to do this or else we could capture more than we want with the embed pattern
```suggestion
    _VALID_URL = r'https?://(?P<server>[\w-]+)\.tv\.pandavideo\.com\.br/embed/?\?(?:[^#"\']*&)?v=(?P<id>[\da-f-]+)'
    _EMBED_REGEX = [rf'<iframe[^>]+\bsrc=["\'](?P<url>{_VALID_URL})']
```